src/core/parse.js

```diff
@@ -1231,7 +1420,7 @@ foam.CLASS({
   package: 'foam.parse',
   name: 'PSymbol',
 
-  properties: [ 'name', 'parser' ]
+  properties: ['name', 'parser']
```

Comment:
I believe we actually do want those spaces to be there.  The linter config I have in the real repo requires those spaces.  I've backported it to this repo (something I should have done a while ago).

I wouldn't worry too much about style in this repo though, we can fix it up when we move stuff to the real repo.